contracts/base/AaveV2Wrapper.sol

```diff
@@ -93,79 +109,140 @@ contract AaveV2Wrapper is OwnableUpgradeable, ReentrancyGuardUpgradeable {
         (, price, , , ) = priceFeed.latestRoundData();
     }
 
-    function normalize(uint256 _amount) internal pure returns (uint256) {
-        return _amount * 10**(ETH_DECIMALS - USD_DECIMALS);
+    /**
+     * @dev Convert given amount to USD decimals
+     * @param _amount asset amount
+     * @param _decimals asset decimals
+     */
+    function toUSDDecimals(uint256 _amount, uint256 _decimals)
```

Comment:
fun tousdDecimals will convert decimal into usd only , or it can used for other asset decimal converter

Reply:
Yeah, it can convert any asset to usd decimals